### ISSUE TITLE
Cannot install on Fedora 24: DB problem (Unknown column 'created' in 'task')

### ISSUE THREAD

**Issue by rm1984** [ansible-semaphore/semaphore#104]
Hi, I'm trying to install Semaphone on a Fedora 24 but I get this error concerning the database:

```
 Running DB Migrations..
Creating migrations table
Executing migration v0.0.0 (at 2016-05-22 07:49:36.108257846 +0200 CEST)...
 [11/11]
Executing migration v0.1.0 (at 2016-05-22 07:49:36.648612822 +0200 CEST)...
 [3/6]
 ERR! Query: alter table task change `created` `created` datetime not null default current_timestamp

Rolling back v0.1.0 (time: 2016-05-22 07:49:36.650500768 +0200 CEST)...
Rollback SQL doesn't exist.


 Database migrations failed!
 Error 1054: Unknown column 'created' in 'task'
```

These are the package versions of Ansible and MariaDB:

```
ansible.noarch                         2.0.2.0-1.fc24
mariadb.x86_64                         1:10.1.14-1.fc24
mariadb-common.x86_64                  1:10.1.14-1.fc24
mariadb-config.x86_64                  1:10.1.14-1.fc24
mariadb-errmsg.x86_64                  1:10.1.14-1.fc24
mariadb-libs.x86_64                    1:10.1.14-1.fc24
mariadb-server.x86_64                  1:10.1.14-1.fc24
```

Thanks! :)

**Comment by matejkramny** [comment]
Hello!

Unfortunately this was a bug introduced by `v2.0.1` for new users (only). It is already fixed, but not released yet.

A temporary fix: setup `v2.0` and then upgrade to `v2.0.1`

**Comment by matejkramny** [comment]
Fixed with v2.0.2
